ARrehab/ARrehab/Minigames/FaceGame/Extras/FoundationExtras.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  FoundationExtras.swift
+//  FaceOff
+//
+//  Created by John Scalo.
+//  Copyright © 2018 Made by Windmill. All rights reserved.
+//
+
+import Foundation
+
+extension Array {
+    func randomItem() -> Element? {
+        if isEmpty { return nil }
+        let index = Int(arc4random_uniform(UInt32(self.count)))
+        return self[index]
+    }
+}
+
+extension Int {
+    static func randomInt(min: Int, max:Int) -> Int {
```

Comment:
https://developer.apple.com/documentation/swift/int/2995648-random
This seems to do the same thing